docs/MoreAboutPersistentStorage.md

```diff
@@ -0,0 +1,44 @@
+| thanos-rule | The thanos ruler evaluates Prometheus recording and alerting rules against chosen query API via repeated query. Rule results are written back to disk in the Prometheus 2.0 storage format. |
+| thanos-receive-default | Thanos receiver accepts incoming data (Prometheus remote-write requests) and writes these into a local instance of the Prometheus TSDB. Periodically (every 2 hours) TSDB blocks are uploaded to the Object storage for long term storage and compaction. The amount of hours/days of data retained in this stateful set (acts a local cache) was fixed in ACM 2.1/2.2. It has been exposed as an API parameter in ACM 2.3: _RetentionInLocal_ |
+| thanos-store-shard| It acts primarily as an API gateway and therefore does not need significant amounts of local disk space. It joins a Thanos cluster on startup and advertises the data it can access. It keeps a small amount of information about all remote blocks on local disk and keeps it in sync with the bucket. This data is generally safe to delete across restarts at the cost of increased startup times. |
+| thanos-store-memcached | Thanos Store Gateway supports an index cache using Memached to speed up postings and series lookups from TSDB blocks indexes. |
+| | |
+
+
+
+
+### Configuring the Stateful sets
+
+In ACM 2.1 and ACM 2.2 only the following was exposed using the Mutlicluster Observability Operator API - _observability.open-cluster-management.io/v1beta1_
+
+```
+    //defaults shown below
+    statefulSetSize: 10Gi
+    statefulSetStorageClass: gp2
+```
+In ACM 2.3, taking into consideration that one size fits all results in wasting space, we have allowed all settings can be individually tweaked in the enhanced API - _observability.open-cluster-management.io/v1beta2_
+
+```
+    //defaults shown below
+    StorageClass: gp2
+	AlertmanagerStorageSize: 1Gi 
+	RuleStorageSize: 1Gi
+	CompactStorageSize: 100 Gi
+	ReceiveStorageSize: 100 Gi
+	StoreStorageSize: 10 Gi
+
+```
+Do note that the default storage class, as configured in the system, is used for configuring the persistent volumes automatically unless a different storage class is specified in the Custom Resource specification. If no storage class exists - for example in default OpenShift bare metal installtions, it will need to be created or the installation of ACM Observability will not succeed.
+
```

Comment:
**Note**: The default storage class, as configured in the system, is used for configuring the persistent volumes automatically unless a different storage class is specified in the custom resource specification. If no storage class exists, for example in default OpenShift bare metal installations, one must be created or the installation of observability fails.